src/pages/wbs-details/wbs-details.tsx

```diff
@@ -3,25 +3,54 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
+import { AxiosResponse } from 'axios';
+import { useEffect, useState } from 'react';
 import { useParams } from 'react-router-dom';
 import { validateWBS, wbsIsProject } from 'utils';
+import { Project } from 'utils/src';
+import ProjectDetails from '../../components/project-details/project-details';
+import { apiFetch } from '../../shared/axios';
 import styles from './wbs-details.module.css';
 
+interface ProjectRequestData {
+  message?: string,
+  data?: Project
+}
+
 const WBSDetails: React.FC = () => {
+  const [projectData, setProjectData] = useState<ProjectRequestData>({});
+
   interface ParamTypes {
     wbsNum: string;
   }
+
   const { wbsNum } = useParams<ParamTypes>();
 
   validateWBS(wbsNum); // ensure the provided wbsNum is correctly formatted
 
+  useEffect(() => {
```

Comment:
this GitHub issue is supposed to just focus on creating the view-only component and not the data fetching aspect because that will be implemented later once all the child components are ready. If you're looking to just make the view-only component available for you to view when running the application, you can just place it hard-coded into the project on a page and provide it one of the dummy data example objects.

if you're interested in the data fetching component, you should take a look at how it is done in other components. we want the design pattern to match as much as possible unless there is a good reason for it to be different. if you have suggestions on how to improve the design pattern for fetching data, I'd be happy to discuss it. see the container components for examples of data fetching.